docs/whats-new.md

```diff
@@ -4,30 +4,49 @@ This page contains highlights of each deck.gl release. Also check our [vis.gl bl
 
 ## deck.gl v9.2
 
-Target release date: July, 2025
+Target release date: September, 2025
+
+### Widgets
+
+- A suite of new widgets have been added to the [`@deck.gl/widgets`](./api-reference/widgets/overview.md) module:
+  - ContextMenuWidget
+  - [FpsWidget](./api-reference/widgets/fps-widget.md)
+  - [GeocoderWidget](./api-reference/widgets/geocoder-widget.md)
+  - GeolocateWidget
+  - [GimbalWidget](./api-reference/widgets/gimbal-widget.md)
+  - [InfoWidget](./api-reference/widgets/info-widget.md)
+  - [LoadingWidget](./api-reference/widgets/loading-widget.md)
+  - [ResetViewWidget](./api-reference/widgets/reset-view-widget.md)
+  - [ScaleWidget](./api-reference/widgets/scale-widget.md)
+  - [ScreenshotWidget](./api-reference/widgets/screenshot-widget.md)
+  - [SplitterWidget](./api-reference/widgets/splitter-widget.md)
+  - StatsWidget
+  - [ThemeWidget](./api-reference/widgets/theme-widget.md)
+  - [TimelineWidget](./api-reference/widgets/timeline-widget.md)
+  - ViewSelectorWidget
+- Pre-wrapped React components - deck.gl widgets are available via the [`@deck.gl/react`](./api-reference/react/overview.md) package
+- Custom widgets in pydeck - via the `custom_libraries` parameter, Python users can integrate custom deck.gl widgets seamlessly.
+- Documentation for styling and custom themes - see [Styling Widgets](./api-reference/widgets/styling).
 
 ### WebGPU Early Preview
 
 A few deck.gl layers can now be run on WebGPU in the website, by selecting the `WebGPU` tab. See documentation about how to [test WebGPU support](./developer-guide/webgpu.md).
 
-### Widgets
+### Core
 
- A suite of new widgets have been added to the [`@deck.gl/widgets`](./api-reference/widgets/overview.md) module:
+- [`View.clone()`](./api-reference/core/view.md) - New method that simplifies creating new Views with modified props, similar to `Layer.clone()`
+- Async DeckPicker methods - New asynchronous methods for improved picking performance and non-blocking operations
+- Multi-view clear support - Enhanced support for controlling clear color, depth, and stencil buffers across multiple views
+- Mapbox v3 Standard style slot support - Integration with Mapbox's latest Standard style slot system for better map styling
 
-- [ResetViewWidget](./api-reference/widgets/reset-view-widget.md)
-- [ScaleWidget](./api-reference/widgets/scale-widget.md)
-- [GeocoderWidget](./api-reference/widgets/geocoder-widget.md)
-- [ScreenshotWidget](./api-reference/widgets/screenshot-widget.md)
-- [LoadingWidget](./api-reference/widgets/loading-widget.md)
-- [ThemeWidget](./api-reference/widgets/theme-widget.md)
-- [InfoWidget](./api-reference/widgets/info-widget.md)
-- [SplitterWidget](./api-reference/widgets/splitter-widget.md)
+### New Layers
```

Comment:
Should we mention the new backgroundBorderRadius TextLayer prop?

AFAIK it was slipped into a 9.1 patch release and is not mentioned in any what's new entry.

To me it would be reasonable to add it to 9.2.